client/src/components/Tags/Tags.js

```diff
@@ -12,7 +12,7 @@ export default class Tags extends React.Component {
         if (!tags) return (<div/>);
         return (
             <div className="tags">
-                {tags.map(tag => {
+                {tags.sort((a, b) => (a.name > b.name) ? 1 : -1).map(tag => {
```

Comment:
speaking of which, you could use that here
```
tags.sort((a, b) => b.name - a.name)
```

Reply:
I'll do that, it's much cleaner.

Reply:
Actually that doesn't seem to work for me.

Reply:
yea I'm dumb, that just returns NaN